desktop-app/app/app-move-to-applications.js

```diff
@@ -0,0 +1,45 @@
+/*
+  conflict handler methods for moveToApplication event on mac
+  https://www.electronjs.org/docs/api/app#appmovetoapplicationsfolderoptions-macos
+*/
+
+// @flow
+
+let conflictMessage = 'An app of this name already exists';
+const moveFailedMessage = 'Something went wrong while moving application';
+
+const confirmMove: boolean = dialog => {
+  const response: number = dialog.showMessageBoxSync({
```

Comment:
Can you please add a `detail` field here with the string `Responsively App should be in the Applications folder to be able to run correctly.`

Reply:
Sure will do that.